src/providers/drift.ts

```diff
@@ -68,23 +62,29 @@ const loadScript = () => {
         })
     }
   })()
+
+  return true
 }
-/* eslint:enable */
+/* eslint-enable */
 
 const load = ({
   providerKey,
   setState
 }: {
   providerKey: string
   setState: (state: State) => void
-}): void => {
-  loadScript()
-  window.drift.load(providerKey)
-  window.drift.SNIPPET_VERSION = '0.3.1'
-  window.drift.on('ready', () => {
-    setState('open')
-    setTimeout(() => setState('complete'), 2000)
-  })
+}): boolean => {
+  const loaded = loadScript()
+
+  if (loaded) {
+    window.drift.load(providerKey)
+    window.drift.SNIPPET_VERSION = '0.3.1'
```

Comment:
Does this lock the drift version? Is it worth a bump to current at the same time?

Reply:
I'm not sure what the current is 🤔